optional-kubernetes-engine/config/php-pubsub.ini

```diff
@@ -1,3 +0,0 @@
-extension=mongo.so
```

Comment:
Why are removing this file? I believe we still want these overrides.

Reply:
`mongo` is not available (let's use `mongodb`), `curl` is built-in, so not needed.

`suhosin`, yeah maybe needed for 5.6, but we pick up 7.1 runtime with the current configuration anyways, so it's not needed unless you modify the composer.json and explicitly allow only php 5.6.